e2e/execution_test.go

```diff
@@ -189,6 +189,35 @@ func testExecution(t *testing.T) {
 		})
 	})
 
+	t.Run("execution with price", func(t *testing.T) {
+		var (
+			inputs = &types.Struct{
+				Fields: map[string]*types.Value{
+					"msg": {
+						Kind: &types.Value_StringValue{
+							StringValue: "test",
+						},
+					},
+				},
+			}
+		)
+		resp, err := client.ExecutionClient.Create(context.Background(), &pb.CreateExecutionRequest{
+			TaskKey:      "task1",
+			EventHash:    hash.Int(1),
+			ExecutorHash: executorHash,
+			Price:        "50atto",
+			Inputs:       inputs,
+		})
+		require.NoError(t, err)
+
+		_, err = streamInProgress.Recv()
+		require.NoError(t, err)
+
+		exec, err := streamCompleted.Recv()
+		require.NoError(t, err)
+		require.Equal(t, resp.Hash, exec.Hash)
```

Comment:
we should check the balances of the accounts ;)

Reply:
Yes, the good e2e tests are missing... It's not only balance, but also service with multiple owners.